wgpu-native/src/binding_model.rs

```diff
@@ -25,6 +25,7 @@ pub enum BindingType {
     Sampler = 1,
     SampledTexture = 2,
     StorageBuffer = 3,
+    UniformBufferDynamic = 4,
```

Comment:
can we add `StorageBufferDynamic` please since we are at it?

Reply:
Certainly!